codeforpdx/PASS
# Bug 454: when clicking on message in inbox the state of the unread me…

# Message Badge Not Updating After Message Read

## This PR:
Resolves #454 
Added the React update to the `<Messages />` page so the state will stay in sync with the podserver message read status for the inbox. 

## Video:
[screen-recorder-mon-oct-23-2023-12-58-19.webm](https://github.com/codeforpdx/PASS/assets/96048/5407c9e0-9234-435c-8831-38f64fb5834c)

## Additional Context (optional):
### How to test

1. Login to podserver and navigate to the messages section
2. send more than one message to a pod you have access to or yourself
3. check that the count is on the message icon on the top right
4. state should update once the podserver is updated


## Issues needing discussion/feedback (optional):
**1.**  Need to add test coverage... could be an additional issue
